src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/OrderItemListDto.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    internal class OrderItemListDto

```

Comment:
This class does not look like a `List` to me?

```suggestion
    internal class OrderItemDto
```